pagopa/io-app
# fix: [PE-605] CGN category list unnecessary loading animation

## Short description
Unnecessary flicks was removed

## List of changes proposed in this pull request
- refactor flattest inside scrollview to flatlist
- add hook for adjusted loading indicator animation

## How to test
- login into the app
- go to cgn card (add it to wallet if necesary)
- go to opportunities
- pressing on any category in the list check the animation

refer to the short video here to get a visual cue https://pagopa.atlassian.net/browse/PE-605
